app/src/main/java/org/keynote/godtools/android/fragment/LanguagesFragment.java

```diff
@@ -120,6 +133,61 @@ public void onDestroyView() {
         super.onDestroyView();
     }
 
+    //region Language Search
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        super.onCreateOptionsMenu(menu, inflater);
+
+        inflater.inflate(R.menu.fragment_language_search, menu);
+
+        // Configuring the SearchView
+        SearchManager searchManager = (SearchManager) getActivity().getSystemService(Context.SEARCH_SERVICE);
+        SearchView searchView = (SearchView) menu.findItem(R.id.app_bar_language_search).getActionView();
+        searchView.setQueryHint(getString(R.string.label_language_search));
+        searchView.setSearchableInfo(searchManager.getSearchableInfo(getActivity().getComponentName()));
+
+        // Will listen for search event and trigger
+        searchView.setOnQueryTextListener(new SearchView.OnQueryTextListener() {
+            @Override
+            public boolean onQueryTextSubmit(String query) {
+                updateLanguageWithSearch(query);
+                return false;
+            }
+
+            @Override
+            public boolean onQueryTextChange(String newText) {
+                updateLanguageWithSearch(newText);
+                return false;
+            }
+        });
+    }
+
+    private void updateLanguageWithSearch(String query) {
+
+        List<Language> queryList = new ArrayList<>();
+
+        // Iterate through list and create new list for adapter
+        if (mLanguages != null) {
+            for(Language language: mLanguages){
+
+                if (language.getDisplayName().toLowerCase().contains(query.toLowerCase())){
+                    queryList.add(language);
+                }
+            }
+        }
+
+        if (mLanguagesAdapter != null) {
+            mLanguagesAdapter.setLanguages(
+                    sortLanguages(queryList)
+            );
+        }
+    }
+
+
```

Comment:
I think checkstyle will also complain about multiple empty lines next to each other